ArticleTemplates/assets/js/bootstraps/liveblog.js

```diff
@@ -312,7 +317,8 @@ define([
             window.liveblogTime();
             window.addEventListener('scroll', util.debounce(updateBlocksOnScroll, 100, true));
             liveMore();
-            
+            creativeInjector.init();
```

Comment:
Now we have the 'trackLiveBlogEpic' exposed from creativeInjector we could actually just call this directly rather than call the 'init'.

By doing this we can remove the logic in creativeInjector.init that checks whether it's a liveblog.